vertexai/multimodal-pdf/pdf_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multimodalpdf
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func Test_generateContentFromPDF(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	buf := new(bytes.Buffer)
+	prompt := `
+		Your are a very professional document summarization specialist.
```

Comment:
suggestion:
```suggestion
	buf := new(bytes.Buffer)
	prompt := `
		You are a very professional document summarization specialist.
```

Reply:
Good catch, thank you

Reply:
Fixing upstream in Python: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/11592